docs/docsite/rst/dev_guide/collections_tech_preview.rst

```diff
@@ -267,6 +267,12 @@ Once you upload a version of a collection, you cannot delete or modify that vers
 uploading. The only way to change a collection is to release a new version. The latest version of a collection (by highest version number)
 will be the version displayed everywhere in Galaxy; however, users will still be able to download older versions.
 
+Collection versions use `Sematic Versioning <https://semver.org/>` for version numbers. Please read the official documentation for details and examples. In summary:
+
+* Increment major (ex. x in `x.y.z`) version number for an incompatible API change.
```

Comment:
change ex. in these three to for example,   or just get rid of that part.  We're trying to eliminate i.e, e.g. ex. etc so that it's clearer and can translate easier.